easyconfigs/m/mne-rsa/mne-rsa-0.8-foss-2022a.eb

```diff
@@ -0,0 +1,30 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonPackage'
+
+name = 'mne-rsa'
+version = '0.8'
+
+homepage = "https://github.com/wmvanvliet/mne-rsa"
+description = """Representational Similarity Analysis on MEG and EEG data"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['5d154c55d382797255d85974797a2059d6b487c26542d82898a6c0ea0ca5ba53']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('scikit-learn', '1.1.2'),
+    ('MNE-Python', '1.3.1'),
+    ('NiBabel', '4.0.2'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+modloadmsg = """Use import mne_rsa"""
```

Comment:
I've built and loaded the module and this useful message isn't very distinct from the other module load messages. Could this be replaced by something like

modloadmsg = """
    Use import %(name)s
    """ 
Or if you preferred single line: 
modloadmsg = "\nUse import %(name)s\n"